geniza/corpus/tests/test_corpus_templates.py

```diff
@@ -58,6 +59,18 @@ def test_multi_viewer(self, client, join):
         second_url = join.textblock_set.last().fragment.iiif_url
         assertContains(response, f'data-iiif-urls="{first_url} {second_url}"')
 
+    def test_edit_link(self, client, document):
+        """Edit link should appear if user is admin, otherwise it should not"""
+        edit_url = reverse("admin:corpus_document_change", args=[document.id])
+        response = client.get(document.get_absolute_url())
+        assertNotContains(response, f'<a class="edit-link" href="{edit_url}">')
```

Comment:
let's just check for the presence of the url, we don't really care what the markup is for this test